tests/test_kharcha.py

```diff
@@ -13,8 +13,6 @@ class TestKharchaAPI(unittest.TestCase):
     def setUp(self):
         self.ctx = app.test_request_context()
         self.ctx.push()
-        db.drop_all()
-        db.create_all()
```

Comment:
We have to drop and create the db structure between tests to ensure there's no overlap between test cases.